# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * scheduler_error_mailer
#
# Translators:
# Niki Waibel <niki.waibel@gmail.com>, 2017
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 10.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-02-08 03:37+0000\n"
"PO-Revision-Date: 2017-02-08 03:37+0000\n"
"Last-Translator: Niki Waibel <niki.waibel@gmail.com>, 2017\n"
"Language-Team: German (https://www.transifex.com/oca/teams/23907/de/)\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: scheduler_error_mailer
#: model:mail.template,body_html:scheduler_error_mailer.scheduler_error_mailer
msgid ""
"\n"
"                \n"
"<div style=\"font-family: 'Lucida Grande', Ubuntu, Arial, Verdana, sans-"
"serif; font-size: 12px; color: rgb(34, 34, 34); background-color: #FFF; \">\n"
"\n"
"<p>Odoo tried to run the scheduler <em>${object.name or ''}</em> in the "
"database <em>${ctx.get('dbname')}</em> but it failed. Here is the error "
"message :</p>\n"
"\n"
"<strong>\n"
"${ctx.get('job_exception') and ctx.get('job_exception').value or 'Failed to "
"get the error message from the context.'}\n"
"</strong>\n"
"\n"
"<p>You may check the logs of the Odoo server to get more information about "
"this failure.</p>\n"
"\n"
"<p>Properties of the scheduler <em>${object.name or ''}</em> :</p>\n"
"<ul>\n"
"<li>Model : ${object.model or ''}</li>\n"
"<li>Method : ${object.function or ''}</li>\n"
"<li>Arguments : ${object.args or ''}</li>\n"
"<li>Interval : ${object.interval_number or '0'} ${object.interval_type or ''}"
"</li>\n"
"<li>Number of calls : ${object.numbercall or '0'}</li>\n"
"<li>Repeat missed : ${object.doall}</li>\n"
"<li>User : ${object.user_id.name or ''}</li>\n"
"</ul>\n"
"\n"
"<p>\n"
"-- <br/>\n"
"Automatic e-mail sent by Odoo. Do not reply.<br/>\n"
"Database : ${ctx.get('dbname')}\n"
"</p>\n"
"</div>\n"
"    \n"
"            "
msgstr ""

#. module: scheduler_error_mailer
#: model:ir.model.fields,field_description:scheduler_error_mailer.field_ir_cron__email_template_id
msgid "Error E-mail Template"
msgstr "Fehler e-Mail Vorlage"

#. module: scheduler_error_mailer
#: model:ir.model,name:scheduler_error_mailer.model_ir_cron
msgid "Scheduled Actions"
msgstr ""

#. module: scheduler_error_mailer
#: model:ir.model.fields,help:scheduler_error_mailer.field_ir_cron__email_template_id
msgid "Select the email template that will be sent when this scheduler fails."
msgstr ""
"WÃ¤hle die e-Mail Vorlage welche bei einem Fehler dieses Schedulers gesendet "
"wird."

#. module: scheduler_error_mailer
#: code:addons/scheduler_error_mailer/models/ir_cron.py:55
#, python-format
msgid "Task failure with UID = %d."
msgstr "Task Fehler von UID = %d."

#. module: scheduler_error_mailer
#: model:ir.actions.server,name:scheduler_error_mailer.test_scheduler_error_mailer_ir_actions_server
#: model:ir.cron,cron_name:scheduler_error_mailer.test_scheduler_error_mailer
#: model:ir.cron,name:scheduler_error_mailer.test_scheduler_error_mailer
msgid "Test Scheduler Error Mailer"
msgstr ""

#. module: scheduler_error_mailer
#: model:mail.template,subject:scheduler_error_mailer.scheduler_error_mailer
msgid "[DB ${ctx.get('dbname')}] Scheduler '${object.name or ''}' FAILED"
msgstr "[DB ${ctx.get('dbname')}] Scheduler '${object.name or ''}' FEHLER"

#, fuzzy
#~ msgid "Email Template"
#~ msgstr "Fehler e-Mail Vorlage"

#~ msgid "ir.cron"
#~ msgstr "ir.cron"
